packages/forms/docs/stories/MultiInput.story.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { Application, Button, GoogleAddressLookup, PhoneInput } from 'react-rainbow-components';
+import ReactJson from 'react-json-view';
+import { Field, UniversalForm } from '@rainbow-modules/forms';
+import MultiInput from '../../src/components/MultiInput';
+
+const GOOGLE_MAPS_APIKEY = process.env.STORYBOOK_GOOGLE_MAPS_APIKEY;
```

Comment:
I checked and we already have an env for this called STORYBOOK_GOOGLE_MAPS_API_KEY in circle CI then let's change it here